subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/ExecuteScheduledTransformationStepBuildOperationDetails.java

```diff
@@ -48,6 +75,11 @@ public String getSubjectName() {
     @Override
     public Object getCustomOperationTraceSerializableModel() {
```

Comment:
Do we still need a `getCustomOperationTraceSerializableModel()` method now that we don't expose Gradle's more complex types?

Reply:
I tried it out and it seems like we still need it.